### ISSUE TITLE
Superiorly placed odontoid process

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9616]
**Preferred term label:**
Superiorly placed odontoid process

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
613686
PMID: 20087400 1/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
It is unclear from the description if this is different from basilar invagination. I do not think the information in the article is enough, and since this is so rare I am closing the issue.
